source/_patterns/02-molecules/vue-widgets/src/vue-clock/vue-clock.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div class="container">
```

Comment:
This a reserved thing in Bootstrap 4 for big layout concepts, components like this should never include `.container`.